admin-base/ui.apps/src/main/js/perAdminApp.js

```diff
@@ -790,7 +790,7 @@ var PerAdminApp = {
      */
     init(perAdminAppView) {
         view = perAdminAppView
-        api = new PeregrineApi(new PerAdminImpl(PerAdminApp))
+        api = new PeregrineApi(new PerAdminImpl(this))
```

Comment:
_Nothing about your code  but I just realised the constructor paramter for `new PerAdminImpl()` is called `callback` which is for me a very confusing name_